app/routes/call.tsx

```diff
@@ -20,30 +34,183 @@ export const loader: LoaderFunction = async ({request}) => {
+          <H6
+            id="episodes"
+            as="h2"
+            className="flex flex-col col-span-full mb-10 lg:flex-row lg:mb-0"
+          >
+            <span>Calls with Kent C. Dodds</span>
+            &nbsp;
+            <span>{` — ${data.episodes.length} episodes`}</span>
+          </H6>
+
+          <button
+            className="text-primary inline-flex items-center text-lg font-medium"
+            onClick={() => setSortOrder(o => (o === 'asc' ? 'desc' : 'asc'))}
+          >
+            {sortOrder === 'asc' ? (
+              <>
+                Showing oldest first
+                <ChevronUpIcon className="ml-2 text-gray-400" />
+              </>
+            ) : (
+              <>
+                Showing newest first
+                <ChevronDownIcon className="ml-2 text-gray-400" />
+              </>
+            )}
+          </button>
+        </div>
+
+        <div className="col-span-full">
+          {sortedEpisodes.map((episode, idx, {length}) => {
+            const number =
+              sortOrder === 'asc' ? idx + 1 : Math.abs(length - idx)
+
+            return (
+              <div
+                className="border-b border-gray-200 dark:border-gray-600"
+                key={episode.slug}
+              >
+                <Link
+                  data-episode={episode.slug}
+                  key={episode.slug}
+                  to={activeSlug === episode.slug ? './' : `./${episode.slug}`}
+                >
+                  <Grid nested className="group relative py-10 lg:py-5">
+                    <div className="bg-secondary absolute -inset-px group-hover:block hidden -mx-6 rounded-lg" />
+                    <div className="relative flex-none col-span-1">
+                      <div className="absolute inset-0 flex items-center justify-center opacity-0 group-hover:opacity-100 transform scale-0 group-hover:scale-100 transition">
+                        <div className="flex-none p-4 text-gray-800 bg-white rounded-full">
+                          <TriangleIcon size={12} />
+                        </div>
+                      </div>
+                      <img
+                        className="w-full rounded-lg object-cover"
+                        src={episode.imageUrl}
+                        alt={episode.title}
+                      />
+                    </div>
+                    <div className="text-primary relative flex flex-col col-span-3 md:col-span-7 lg:flex-row lg:col-span-11 lg:items-center lg:justify-between">
+                      <div className="mb-3 text-xl font-medium lg:mb-0">
+                        {/* TODO: make it three digits? How many calls do we expect? */}
+                        <span className="inline-block w-10 lg:text-lg">
+                          {`${number.toString().padStart(2, '0')}.`}
+                        </span>
+
+                        {episode.title}
```

Comment:
Could/should we also display the episode summary and keywords in a small font here? I'm not sure about the summary, but keywords might be useful. The problem is, I'm not sure what the summary will be. Right now we don't connect that information anyway...

Reply:
With summary you mean `description` right? There is no limit on how long that can get. That's why I showed it inside the accordion.

I'll add the keywords. But there also, there is no limit on how many. Should we stick to the first 3? Separated by space?

Reply:
Maybe even just add both in the accordion. Looks quite nice!

![image](https://user-images.githubusercontent.com/1196524/126530596-52155b4c-8921-4204-adf5-ff6452fa8db8.png)

Reply:
Why don't we put the keywords (unlabeled) in muted text below the title? I just want to make it easy to scan the episodes. Maybe that's not necessary though 🤔

Reply:
I've created #77. Let's take another look at that asap.